### DISCUSSION TITLE
Question about onunload() event and Dropdown block?

### DISCUSSION THREAD

**Discussion by wantingjun** [lowdefy/lowdefy#1042] [Questions and Help]
Hi,

I have some questions about lowdefy

### 1. onunload() or onbeforeunload() events
I want to listen for the tab or browsers closing event and do some operations in this event, there are actually have some api in JavaScript, like `onunload()` or `onbeforeunload()`, but I haven't found any in lowdefy. I am wondering whether Lowdefy has provided the operator (Maybe I just missed it)? And if the Lowdefy do not provide, what can i do to achieve my goal?

### 2. dropdown component
[dropdown component ](https://ant.design/components/dropdown/)
![image](https://user-images.githubusercontent.com/43306263/148151907-d2114b59-8074-4787-b2c5-d95fa2f0640a.png)

whether the Lowdefy has the component like the dropdown?

Thank you for your help.

Tingjun

**Comment by Gervwyk** [comment]
Hi @wantingjun 

We have not currently implemented the Dropdown component, but it would make sense to add it. Adding a block into the Lowdefy antd library is an easy process. If you would like to contribute and write the code for this that would be great. It's as simple as adding React component. Else, I'll add a feature issue for it to be included in the v4 release.
